VideoWeb/VideoWeb/Controllers/ConferencesController.cs

```diff
@@ -254,23 +254,15 @@ public async Task<ActionResult<ConferenceResponse>> GetConferenceById(Guid confe
             return Ok(response);
         }
 
-        private static void ValidateConferenceAndBookingParticipantsMatch(List<ParticipantDetailsResponse> participants,
-            List<BookingParticipant> bookingParticipants)
+        private static void ValidateConferenceAndBookingParticipantsMatch(IEnumerable<ParticipantDetailsResponse> participants,
+            IReadOnlyCollection<BookingParticipant> bookingParticipants)
         {
-            List<Exception> missingBookingParticipantIds = new List<Exception>();
-            foreach (var participant in participants)
-            {
-                if (bookingParticipants.SingleOrDefault(p => p.Id == participant.Ref_id) == null)
-                {
-                    missingBookingParticipantIds.Add(new ArgumentNullException(
-                        $"Unable to find a participant in bookings api with id ${participant.Ref_id}"));
-                }
-            }
+            var missingBookingParticipantIds = (from participant in participants where bookingParticipants.SingleOrDefault(p => p.Id == participant.Ref_id) == null select new ArgumentNullException($"Unable to find a participant in bookings api with id ${participant.Ref_id}")).Cast<Exception>().ToList();
```

Comment:
what was the reasoning to change this? it's unrelated to the story and makes it harder to read what is happening here

Reply:
Nick could you please answer, I believed it's your change

Reply:
@npoxon could you revert this change please? It makes for a harder read

Reply:
@shaed-parkar that is Resharper. Will revert it, 2 mins

Reply:
@shaed-parkar @valentinabennett Done. Linq was being a little too clever. Pushed